hack/generator/pkg/astmodel/types.go

```diff
@@ -204,3 +203,22 @@ func (types Types) ResolveResourceStatusDefinition(
 	// preserve outer spec name
 	return resourceStatusDef.WithName(statusName), nil
 }
+
+// Process applies a func to transform all members of this set of type definitions, returning a new set of type
+// definitions containing the results of the transfomration, or possibly an error
```

Comment:
```suggestion
// definitions containing the results of the transformation, or possibly an error
```

Reply:
Missed this fix - included in my current branch instead.